api/src/test/java/org/openmrs/module/bacteriology/api/impl/BacteriologyServiceTest.java

```diff
@@ -324,7 +324,7 @@ public void ensureGetSpecimenReturnsCorrectSpecimen() throws Exception{
 
         Specimen specimen = bacteriologyService.getSpecimen(obsGroup);
         assertEquals("SAMPLE12345",specimen.getIdentifier());
-        assertNull(specimen.getReport());
+        assertNotNull(specimen.getReport());
```

Comment:
any particular reason the assertion is changed in this case?

Reply:
As the test data for 100 object has changed as part of this commit(to fix the IT), the specific specimen has report data. So it's no longer null for the specific test data.

Reply:
As the test data for 100 object has changed as part of this commit(to fix the IT), the specific specimen has report data. So it's no longer null for the specific test data.